example_test.go

```diff
@@ -3,7 +3,10 @@ package base36_test
 import (
 	"fmt"
 
-	"github.com/martinlindhe/base36"
+	// change depending on which github account the package belongs to 
+	//"github.com/martinlindhe/base36"
+	"github.com/buzzfrog/base36"
```

Comment:
This makes no sense, please fix to use the proper import path and drop the comment.